client/core/types.go

```diff
@@ -695,6 +695,18 @@ type OrderFilter struct {
 	Statuses []order.OrderStatus `json:"statuses"`
 }
 
+// Account holds data returned from AccountExport.
+type Account struct {
+	Host      string `json:"host"`
+	AccountID string `json:"accountID"`
+	PrivKey   string `json:"privKey"`
+	PubKey    string `json:"pubKey"`
+	EncKey    string `json:"encKey"`
```

Comment:
I'm not certain we want to include `EncKey`.  For a given app password, `EncKey` is derived from `PrivKey`.